botocore/docs/__init__.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import os
+
+import botocore.session
+from botocore.compat import six
+from botocore.docs.service import ServiceDocumenter
+
+
+def generate_docs():
```

Comment:
I would suggest adding a parameter that lets you specify the output directory.  That way we don't depend on the fact that we can right to `<root>../docs` and we can also just have the docs/conf.py specify the directory botocore should generate documentation.  Also, we won't have to use a patch in test_docs.py.